Editor/Analytics/AnalyticsUtils.cs

```diff
@@ -25,5 +25,20 @@ internal static void SetTrackingTo(AnalyticsState state)
         {
             EditorPrefExtensions.SetEnum(KeyTrackingState, state);
         }
+
+        internal static void ShowDataPrivacyStatement()
+        {
+            string pathToDataPrivacyStatement = "/.Documentation/articles/innoactive-creator/data-privacy-information.md";
+            if (EditorUtils.GetCoreVersion().Equals("unknown"))
+            {
+                string dataPrivacyStatementUrl = EditorUtils.GetCoreFolder() + pathToDataPrivacyStatement;
+                UnityEditorInternal.InternalEditorUtility.OpenFileAtLineExternal(dataPrivacyStatementUrl, 1);
```

Comment:
Does this work for you? This method is not in the public API and to me (using MacOS) it only opens Rider but it never shows the file 😕

Reply:
Yeah it works for me. I had a "which program do you want to use to open the file" dialog in the beginning and now it always opens VS Code to show it